### ISSUE TITLE
Log message format

### ISSUE THREAD

**Issue by klauer** [pcdshub/lcls-twincat-general#12]
While we can go with plaintext and worry about filtering down the line, it seems like it might be a good idea to tackle log message formatting from the start and include some structure.

Syslog itself has a few spots where we can insert information to better categorize messages, such as PID or message ID, which only need to be ASCII. Beyond that, maybe we could consider using JSON in the message with some versioned schema:

```
{
"schema": "plc-general-v1",
"msg": "message text",
"schema_metadata1": 123, 
"schema_metadata2": 456, 
...
}
```

Fields in logstash [can contain JSON](https://www.elastic.co/guide/en/logstash/current/plugins-filters-json.html), which would be expanded either into the root event or nested somewhere.

**Comment by klauer** [comment]
Adding onto the above, there's nothing in logstash requiring us to even use the syslog format.

We could jump to JSON-only if desirable, removing the need of shoehorning in information to fields that don't match 100% to what the specification is saying.

**Comment by klauer** [comment]
Addressed by #14. Schema can be revisited later. Closing.
